src/infra/tracing/tracing.rs

```diff
@@ -123,23 +123,26 @@ pub async fn init_tracing(config: &TracingConfig, sentry_url: Option<&str>, toki
         }
     };
 
-    // configure tokio-console layer
-    println!("tracing registry: enabling tokio console exporter | address={}", tokio_console_address);
-    let (console_layer, console_server) = ConsoleLayer::builder().with_default_env().server_addr(tokio_console_address).build();
-    spawn_named("console::grpc-server", async move {
-        if let Err(e) = console_server.serve().await {
-            tracing::error!(reason = ?e, address = %tokio_console_address, "failed to create tokio-console server");
-        };
-    });
-
-    // init registry
-    let result = tracing_subscriber::registry()
```

Comment:
No need to change the registry initialization.

Follow the same pattern used in opentelemetry initialization where it returns a None layer when it is disabled.